### PR TITLE
Configure all identifiers in a single json file

### LINKED ISSUE #292: Configure identifiers in a single place
Identifiers are currently configured in the migrations, the InitializeSystem script, and the PublishPrices script. We should create a universal configuration file for these that can change the identifiers based on network.

### PR DESCRIPTION
**Pull request by ptare**
React doesn't seem to allow imports outside of src/, and this PR adds a symlink from within the src/ directory. Also if you symlink to JS, that JS doesn't go through Babel properly, so the target has to be a json file.

identifiersTest.json is a config that works with only AlphaVantage products.

Fixes #292

### CODE REVIEW

**Comment by mrice32** [comment]
I assume this fixes #292?

File: sponsor-dapp/src/App.js [hunk 1]
```diff
@@ -79,7 +80,7 @@ class App extends Component {
             if (this.props.params) {
               newParams = { ...this.props.params };
             } else {
-              newParams = { ...params };
+              newParams = { ...params, identifiers: { ...identifierConfig } };
```
Comment (mrice32) [inline sponsor-dapp/src/App.js]: As discussed IRL, could we make the config.json file a .js file and import/append identifiers.json within that file?

File: sponsor-dapp/src/components/CreateContractModal.js [hunk 2]
```diff
@@ -100,13 +100,13 @@ class CreateContractModal extends React.Component {
     const constructorParams = {
       sponsor: account,
       defaultPenalty: web3.utils.toWei("1", "ether"),
-      supportedMove: web3.utils.toWei(params.identifiers[formInputs.identifier].supportedMove, "ether"),
+      supportedMove: web3.utils.toWei(params.identifiers[formInputs.identifier].dappConfig.supportedMove, "ether"),
```
Comment (mrice32) [inline sponsor-dapp/src/components/CreateContractModal.js]: Do we need the entire `identifiers.json` config or could we move the `dappConfig` part up a level and ignore all the other subobjects?

File: sponsor-dapp/src/App.js (hunk 1 above)
Comment (ptare) [reply sponsor-dapp/src/App.js]: Makes a ton of sense to me, done.

File: sponsor-dapp/src/components/CreateContractModal.js (hunk 2 above)
Comment (ptare) [reply sponsor-dapp/src/components/CreateContractModal.js]: Done.

**Pull request merged**